libs/react-helpers/src/lib/format/number.ts

```diff
@@ -32,13 +36,14 @@ export function removeDecimal(value: string, decimals: number): string {
 }
 
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat
-export const getNumberFormat = memoize(
-  (digits: number) =>
-    new Intl.NumberFormat(getUserLocale(), {
-      minimumFractionDigits: digits,
-      maximumFractionDigits: digits,
-    })
-);
+export const getNumberFormat = memoize((digits: number) => {
+  if (isNil(digits) || digits < 0)
```

Comment:
chore (blocking):  braces here please

Reply:
Added a eslint rule for this https://github.com/vegaprotocol/frontend-monorepo/pull/1322

Reply:
updated.